src/main/java/access2csv/Driver.java

```diff
@@ -129,25 +140,44 @@ public static void main(String[] args) throws Exception {
 			parser.printHelpOn(System.out);
 			return;
 		}
-
+      
 		File inputFile = input.value(options);
 		if(!inputFile.exists()) {
 			throw new FileNotFoundException("Could not find input file: " + inputFile.toString());
 		}
 		
-		File outputDir = output.value(options);
-		if(!outputDir.exists()) {
-			outputDir.mkdirs();
+		File outputDir = null;
+		if (options.has(output)) {
+			outputDir = output.value(options);
+			if(!outputDir.exists()) {
+				outputDir.mkdirs();
+			}	
 		}
 		
-		if(options.has(schema)) {
-			exportAll(inputFile, options.has(withHeader), outputDir, csvPrefix.value(options));
+		boolean applyQuotesToAll = true;
+			if (options.has(quoteAll)) {
+			applyQuotesToAll = quoteAll.value(options);
 		}
-		else if (options.has(table)){
-			export(inputFile, table.value(options), outputDir, csvPrefix.value(options));
+    
+		String nullText = null;
+		if (options.has(writeNull)) {
```

Comment:
do we need the `has` check? I think the default value lets us unconditionally use `String nullText = writeNull.value(options);`

Reply:
I'll look at it again.

Reply:
I think writeNull would be null if it is not included in the command line arguments. I'll try it out first.

Reply:
You were right about this one too. I will update the pull request. Thank you!